packages/mobile/src/app/selectors.ts

```diff
@@ -218,7 +218,16 @@ export const registrationStepsSelector = createSelector(
       }
     }
     if (skipVerification) {
-      totalSteps--
+      if (Object.keys(steps).includes(Screens.VerificationEducationScreen)) {
+        totalSteps--
+        step =
+          step >
```

Comment:
```suggestion
          step >=
```
I think this needs to be >=

Reply:
not 100% sure

Reply:
Equal to would be the actual VerificationEducationScreen, which should hopefully never be shown when skipVerification is true